### PR TITLE
Improve UX

### PR DESCRIPTION
**Pull request by thomas-tosoni**
Row Middle Clicks
Ad Layout File Upload

### CODE REVIEW

**Comment by dduponchel** [comment]
This pull request generates heated discussions, we'll need @stephane-mics in the loop.

**Comment by thomas-tosoni** [comment]
Now that we agreed, PR can be merged

**Comment by dduponchel** [comment]
To maximize the clickable area, you should set the padding on the `td` to `0` and set the padding of the link to `$table-cell-padding` (beware of the class `clickableRow`).

File: app/src/core/goals/view.all.html [hunk 1]
```diff
@@ -23,10 +23,10 @@
       </tr>
       </thead>
       <tbody>
-      <tr ng-click="showGoalReport(goal)" ng-repeat="goal in goals">
-        <td><a>{{goal.name}}</a></td>
+      <tr ng-repeat="goal in goals">
+        <td><a class="clickable-box" ng-href="#{{getGoalReportUrl(goal)}}">{{goal.name}}</a></td>
```
Comment (dduponchel) [inline app/src/core/goals/view.all.html]: The `clickable-box` class should be on the `<td>`, not on the `<a>`.

**Pull request merged**